cactusbot/services/beam/handler.py

```diff
@@ -79,17 +77,23 @@ def __init__(self, channel):
     async def handle_liveloading(self, packet):
```

Comment:
If you could change `*liveloading*` to `*constellation*`, that'd be great. (`handle_constellation`, `constellation_events`, etc.) :green_heart: